src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -189,7 +189,10 @@ protected function describeContainerDefinition(Definition $definition, array $op
         $output .= "\n".'- Abstract: '.($definition->isAbstract() ? 'yes' : 'no');
 
         if (method_exists($definition, 'isAutowired')) {
```

Comment:
No problem, I can open one. I just wanted to be sure I didn't miss something. It seems the same happened for the `shared` property. 
Thank you. :)